addons/mail/static/src/views/fields/many2many_avatar_user_field/many2many_avatar_user_field.scss

```diff
@@ -1,4 +1,4 @@
-.o_kanban_renderer, .o_list_renderer {
+.o_kanban_renderer, .o_list_renderer, .o_activity_view {
```

Comment:
This specificity seems annoying, can't we remove this top level selector instead?

Reply:
I think it is just to have that specific style in three specific views and not in the other ones. 🤔